tests/repo_origin/.git-orig/hooks/fsmonitor-watchman.sample

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/perl
```

Comment:
I think you are just trying to out lines added me at this point.  I am unable to provide any form of a review for this.